onnx/common/ir.h

```diff
@@ -866,6 +866,50 @@ friend struct Value;
 
   std::vector <OpSetID> opset_versions_;
 
+  bool isNameUnique(const std::string& name) const {
```

Comment:
isNameUnique == true logically suggests that the function is going to return true if name is unique and false otherwise but the return value of this function is opposite of this and is confusing.

Reply:
my bad 😂